fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -793,10 +791,10 @@ private static void checkLinkForLdpResourceCreation(final String link) {
     private static String parseDigestHeader(final String digest) throws InvalidChecksumException {
         try {
             final Map<String,String> digestPairs = RFC3230_SPLITTER.split(nullToEmpty(digest));
-            final Set<String> checksumTypes =
-                    digestPairs.keySet().stream().map(s -> s.toLowerCase()).collect(Collectors.toSet());
+            final boolean checksumTypeIncludeSHA1 = digestPairs.keySet().stream().map(String::toLowerCase)
+                .anyMatch("sha1"::equals);
```

Comment:
Oh yeah, I kept looking for case insensitive matching in the `Set<String>` when I started this. Now that we are back to just using `String` I never thought about those useful methods. 👍